openshift-storage-libs/openshiftstoragelibs/heketi_ops.py

```diff
@@ -100,13 +100,14 @@ def _set_heketi_global_flags(heketi_server_url, **kwargs):
 
 
 def heketi_volume_create(heketi_client_node, heketi_server_url, size,
-                         raise_on_error=True, **kwargs):
+                         timeout=None, raise_on_error=True, **kwargs):
     """Creates heketi volume with the given user options.
 
     Args:
         heketi_client_node (str): Node on which cmd has to be executed.
         heketi_server_url (str): Heketi server url
         size (str): Volume size
+        timeout(int): Timeout value for heketi volume create cmd
```

Comment:
can we have a default timeout value mentioned in args *HEKETI_COMMAND_TIMEOUT*

Reply:
`HEKETI_COMMAND_TIMEOUT` is a global variable, which default is 120 sec, which is used for all the around the code base, but for this need to have max out time, if we pass `timeout` then it will change the default otherwise it will continue with the default 120sec

Reply:
*timeout=None* parameter in line 103, shows that the timeout value is passed as none, this is to let others know that the default value is 120 and to change this timeout value only the cases that require the timeout value to be changed.
If not you will see the future test cases using this method will have a timeout parameter present.

Reply:
Done